test/e2e/gitopsaddon/scripts/setup_env.sh

```
@@ -203,6 +206,13 @@ spec:
 echo "${ROUTE_CRD}" | ${KUBECTL} --context ${SPOKE_CTX} apply --server-side --force-conflicts -f -
 echo "${ROUTE_CRD}" | ${KUBECTL} --context ${HUB_CTX} apply --server-side --force-conflicts -f -
 
+# Restart the ArgoCD operator so it discovers the Route CRD. Without this, the operator
+# fails to reconcile ArgoCD CRs in non-default namespaces (e.g. local-cluster) with
+# "no kind is registered for the type v1.Route".
+echo ">> Restarting ArgoCD operator to pick up Route CRD"
+${KUBECTL} --context ${HUB_CTX} rollout restart deployment argocd-operator-controller-manager -n argocd-operator-system
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)